### ISSUE TITLE
Multithreading in `apply_forrest`

### ISSUE THREAD

**Issue by pfarndt** [JuliaAI/DecisionTree.jl#134]
I just did some test and realized that when using many threads, e.g. 32, and handling RandomForestRegressors with many trees, e.g. 1000, it is way faster to build a forest than to apply a forest to get a prediction.

I think this is due to the fact that multiple threads are used to build the forrest in parallel, however the "application" of a forest is done sequentially. Would it be conceivable to also apply a forest using multiple threads?

This this issue could also be related to [this issue](https://github.com/bensadeghi/DecisionTree.jl/issues/15)

**Comment by ablaom** [comment]
Multithreading in prediction was added in https://github.com/JuliaAI/DecisionTree.jl/pull/188/files . You need to buy in by specifying `use_multithreading=false` in `apply_forest(...)`. 

Closing, but feel free to open a new issue if you are still seeing issues.
